deploy.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+echo "Bumping version number."
+VERSION=$(grep -o "VERSION @\"[0-9]*\.[0-9]*\.[0-9]*" Mixpanel/Mixpanel.m | tail -1 | tr -d 'VERSION @"')
+VERSION_MAJOR=$(echo $VERSION | awk -F \. '{print $1}')
+VERSION_MINOR=$(echo $VERSION | awk -F \. '{print $2}')
+VERSION_PATCH=$(echo $VERSION | awk -F \. '{print $3}')
+
+if [ $VERSION_PATCH ==  9 ]; then
+    if [ $VERSION_MINOR ==  9 ]; then
```

Comment:
Similar tools take `--major`/`--minor`/`--patch` style command-line arguments to determine how to bump.